### ISSUE TITLE
STRUCTURE - How to create PR for non-developers

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-docs#211]
We need a section of docs that will walk them through creating a PR.

**Comment by mkrzewinski** [comment]
I agree, the Lookit tutorial is good, but I feel that it can be more condensed.

**Comment by becky-gilbert** [comment]
It turns out this section exists! It's in the tutorial: https://lookit.readthedocs.io/en/develop/tutorial-contributing.html#making-your-first-pull-request-pr
And there's a more developer-focused version here: https://lookit.readthedocs.io/en/master/contributor-guidelines.html#add-your-own-feature-and-submit-a-pull-request

@okaycj I agree that the tutorial section doesn't need much work. Here are a few things I noticed while reading through the contributor section about creating PRs:

1. Maybe somewhere at the top, we should reference the tutorial PR section for people who are new-ish to github and just want a refresher/reminder of the process. (It looks like we're already referencing another page for higher level GH workflow stuff: "At a high level, we are roughly following a Forking Workflow version of Gitflow as described here.", and the tutorial PR section does already reference the contributor PR section).

2. Even though the example uses the lookit-api repo, I think this section is intended to be relevant/helpful for all repos. The Getting Started section mentions all repos/branches, and PR section does point out how to do things in lookit-api vs EFP:

```
On lookit-api, you should then update dependencies like this:
... etc ...
On ember-lookit-frameplayer, you should update dependencies using the package manager yarn.
... etc ...
```

So we may want to re-read this section and see if we need to reference anything specific about the other repos (e.g. lookit-jspsych and lookit-docs in the part about updating dependencies above).

3. The paragraph that starts with "Prior to finalizing your commit, make sure to clean up your code to comply with PEP8." seems specific to python and lookit-api, so maybe we should just flag that at the start ("If you're contributing to the lookit-api, then prior to finalizing..."). Or we could have some subsections for contributing to the different repos, if you think that would be clearer.

4. Maybe we need an admonition box with reminders about the main problems we run into: (1) keeping your fork/branch up to date, (2) merging/rebasing upstream and fixing conflicts, and (3) keeping your branch/PR very small and specific? That stuff is in the text but it's kind of buried.
